### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by nllong**
#### What does this PR accomplish?

Change log before release. Please add any additional items you see fit. Add by the way of GitHub suggestions if possible.

#### How should this be manually tested?
n/a

#### What are the relevant tickets?
#194

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,8 +1,20 @@
 Change Log
 ==========
 
+Version 0.2.1
+-------------
+* New command line interface (CLI) for scaffolding project using results of URBANopt SDK's OpenStudio results
+* New script for converting CSV file into Modelica mos file
+* Cleanup of System Parameter Schema including renaming elements, adding definitions, and adding units
+* Redesign the couplings and remove redundant model connector files
+* Promote DES configuration variables to be accessible in the System Parameter file
```
Comment (amyeallen1) [inline CHANGELOG.rst:10]: + * Extended flexibility of setting parameter values programmatically for Teaser, TimeSeries, and Spawn building load models models

**Review by amyeallen1 — COMMENTED** [review]
Thanks, Nick! I added a suggestion to describe the de-hardcoding if you think that's appropriate to include.

File: CHANGELOG.rst (hunk 1 above)
Comment (nllong) [reply CHANGELOG.rst:10]: included, thanks!

**Review by vtnate — APPROVED** [review]
🃏

**Review by macintoshpie — APPROVED** [review]
LGTM 🐞

**Comment by nllong** [comment]
Do y'all just try to find the most random emojis? 🧨

**Pull request merged**